src/components/Router.tsx

```diff
@@ -5,27 +5,31 @@ import { Clusters, ClusterPage, NewClusterPage } from './clusters';
 import { store } from '../store';
 import { isSingleClusterMode, routeBasePath } from '../config/constants';
 import SingleCluster from './SingleCluster';
+import { AssistedUILibVersion } from './ui';
 
 export const AssistedUiRouter: React.FC = () => (
   <Provider store={store}>
     <Router />
   </Provider>
 );
 
-export const Router: React.FC = ({ children }) =>
-  isSingleClusterMode() ? (
-    <Switch>
-      <Route path={`${routeBasePath}/clusters/:clusterId`} component={ClusterPage} />
-      <Route path={`${routeBasePath}/clusters`} component={SingleCluster} />
-      {children}
-      <Redirect to={`${routeBasePath}/clusters`} />
-    </Switch>
-  ) : (
-    <Switch>
-      <Route path={`${routeBasePath}/clusters/~new`} component={NewClusterPage} />
-      <Route path={`${routeBasePath}/clusters/:clusterId`} component={ClusterPage} />
-      <Route path={`${routeBasePath}/clusters`} component={Clusters} />
-      {children}
-      <Redirect to={`${routeBasePath}/clusters`} />
-    </Switch>
-  );
+export const Router: React.FC = ({ children }) => (
+  <AssistedUILibVersion>
```

Comment:
Is it necessary for the component to wrap the contents of a router? The component will need to be placed somewhere else in OCM once the cluster list/detail integration work is finished.

Reply:
Just a short-cut. I will move it somewhere down.